src/DropDownMenu/DropDownMenu.js

```diff
@@ -114,6 +114,11 @@ class DropDownMenu extends Component {
      */
     menuStyle: PropTypes.object,
     /**
+     * If true, `value` must be an array and the menu will support
+     * multiple selections.
+     */
+    multiple: PropTypes.bool,
+    /**
```

Comment:
Is this prop useful outside of `SelectField`, or can it be `@ignore`d for the public `DropDownMenu` API docs?